com.lablicate.OpenChrom.yaml

```diff
@@ -0,0 +1,49 @@
+app-id: com.lablicate.OpenChrom
+runtime: org.freedesktop.Platform
+runtime-version: 23.08
+sdk: org.freedesktop.Sdk
+command: openchrom
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --persist=.openchrom
+  - --persist=.eclipseswtchartsettings
+  - --filesystem=/tmp:rw

```

Comment:
Why is this needed? Can't you set the TMPDIR env var?

Reply:
Taken from https://github.com/flathub/org.eclipse.Java/blob/49768643348b2efcfa330eb0fe41df5317ba9d21/org.eclipse.Java.json#L11 it is required for the p2 update mechanism.